src/Illuminate/Console/ConfirmableTrait.php

```diff
@@ -13,13 +13,12 @@ trait ConfirmableTrait
      * @param  \Closure|bool|null  $callback
      * @return bool
      */
-    public function confirmToProceed($warning = 'Application In Production!', $callback = null)
+    public function confirmToProceed($warning = null, $callback = null)
```

Comment:
This is a breaking change.

Reply:
@driesvints I'll try to revise by keeping this.

Reply:
@driesvints Are you 100% sure this is a breaking change?  If you look at the logic below, this message is only previous valid if $shouldConfirm comes back as true.  I have changed some of the structure but the resulting logic should not be any different.

Reply:
Yes

Reply:
@driesvints Ok.  I'll try to think through this a bit more and revise shortly.  Thanks!

Reply:
@driesvints I walked back that change.  Should be ok now.  Ultimately it might make sense for this to be a separate trait or method altogether.  Not quite sure how to get it there without doing something more extreme but this should at least work / solve the concern.  Let me know what you think.